tests/ctst/common/hooks.ts

```diff
@@ -88,4 +88,9 @@ After({ tags: '@BP-ASSUME_ROLE_USER_CROSS_ACCOUNT'}, async function (this: Zenko
     await cleanupAccount(this, crossAccountName);
 });
 
+After({ tags: '@SSL'}, async function (this: Zenko) {
```

Comment:
should we not do this every time, to ensure it gets reset even if the `@SSL` is (by mistake) not set?